docs/getting-started.md

```diff
@@ -0,0 +1,184 @@
+# Getting Started
+
+* TOC
+{:toc}
+
+These guide(s) will show you how to get started with your favourite container orchestrator as well as Kedge!
+
+There are three different guides depending on your container orchestrator as well as operating system.
+
+For beginners and the most compatibility, follow the __Minikube and Kedge__ guide.
+
+## Minikube and Kedge
+
+In this guide, we'll deploy a sample kedge file `httpd.yaml` to a Kubernetes cluster.
+
+Requirements:
+  - [minikube](https://github.com/kubernetes/minikube)
+  - [kedge](https://github.com/kedgeproject/kedge)
+
+__Start `minikube`:__
+
+If you don't already have a Kubernetes cluster running, [minikube](https://github.com/kubernetes/minikube) is the best way to get started.
+
+```sh
+$ minikube start
+Starting local Kubernetes v1.7.5 cluster...
+Starting VM...
+Getting VM IP address...
+Moving files into cluster...
+Setting up certs...
+Connecting to cluster...
+Setting up kubeconfig...
+Starting cluster components...
+Kubectl is now configured to use the cluster
+```
+
+__Download the [httpd.yaml](https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/httpd/httpd.yaml) example file, or another example from the [examples](https://github.com/kedgeproject/kedge/tree/master/examples) GitHub directory:__
+
+```sh
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/httpd/httpd.yaml
```

Comment:
nit: `wget` is not installed in lot of machines by default, how about `curl -O`

Reply:
You're right. Added!